zulipterminal/model.py

```diff
@@ -301,10 +303,34 @@ def set_narrow(
                 break
         else:
             raise RuntimeError("Model.set_narrow parameters used incorrectly.")
+        
+        try:
+            if self.stream_id:
+                self.active_button = self.controller.view.stream_id_to_button[
+                    self.stream_id
+                ]
+        except AttributeError:
+            pass
 
         if new_narrow != self.narrow:
+            
+            try:
+                if self.stream_id and new_narrow:
+                    self.active_button.set_status(is_active=False)
+            except AttributeError:
+                pass
```

Comment:
Why is everything now wrapped in try/except blocks?

Reply:
I was going to ask for why these errors were popping up, but until then I decided to wrap them in try/except. I'll have to see why it's happening.

Reply:
I think this can overall be much simpler if you think about the logic in the following sections.